AssertionError: assert($form_display instanceof EntityFormDisplayInterface) line 64 localgov_events.module

Just a quick note that I'm seeing this on install:

```
AssertionError: assert($form_display instanceof EntityFormDisplayInterface) in assert() (line 64 of /app/web/modules/contrib/localgov_events/localgov_events.module) #0 /app/web/modules/contrib/localgov_events/localgov_events.module(64): assert(false, 'assert($form_di...')
#1 /app/web/modules/contrib/localgov_events/localgov_events.install(18): localgov_events_optional_fields_settings(true, true)
#2 [internal function]: localgov_events_install(true)
#3 /app/web/core/lib/Drupal/Core/Extension/ModuleHandler.php(392): call_user_func_array('localgov_events...', Array)
```

Looks like these fail assertion as they don't exist yet.

--- a/localgov_events.install
+++ b/localgov_events.install
@@ -10,7 +10,12 @@ use Drupal\taxonomy\Entity\Term;
 /**
  * Implements hook_install().
  */
-function localgov_events_install() {
+function localgov_events_install($is_syncing) {
+
+  // Don't configure the extra fields or config during config sync operations.
+  if ($is_syncing) {
+    return;
+  }
 
   // Configure optional fields.
   $directory_page = \Drupal::moduleHandler()->moduleExists('localgov_directories_page');

--- a/localgov_events.module
+++ b/localgov_events.module
@@ -92,7 +92,13 @@ function localgov_events_page_attachments(array &$attachments) {
 /**
  * Implements hook_modules_installed().
  */
-function localgov_events_modules_installed($modules) {
+function localgov_events_modules_installed($modules, $is_syncing) {
+
+  // Don't configure the extra fields during config sync operations.
+  if ($is_syncing) {
+    return;
+  }
+
   // Configure optional fields.
   $directory_page = in_array('localgov_directories_page', $modules);
   $directory_venue = in_array('localgov_directories_venue', $modules);

